node/service/src/chain_spec/moonbase.rs

```diff
@@ -167,156 +159,6 @@ pub fn get_chain_spec(para_id: ParaId) -> ChainSpec {
 	.build()
```

Comment:
We should use `.with_genesis_config_preset_name(sp_genesis_builder::DEV_RUNTIME_PRESET)` here, instead of `.with_genesis_config(testnet_genesis(`.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3138/commits/e61473f1780b1a8d53fb179662c5c3f611a6aa6a